cpp/test/sg/cd_test.cu

```diff
@@ -72,6 +76,16 @@ class CdTest : public ::testing::TestWithParam<CdInputs<T>> {
     T labels_h[params.n_row] = {6.0, 8.3, 9.8, 11.2};
     raft::update_device(labels.data(), labels_h, params.n_row, stream);
 
+    /* How to get the coefficients:
+
+    from sklearn.preprocessing import StandardScaler
+    scaler = StandardScaler(with_mean=True, with_std=True)
+    x_norm = scaler.fit_transform(data_h)
```

Comment:
Is this how you computed the reference coefficients for the test assertions? If so, can you change it to somehting like "How to reproduce the coefficients for this test".

Reply:
Sure, thanks!